notifications.bs

```diff
@@ -541,18 +557,53 @@ interpreted as a language tag. Validity or well-formedness are not enforced. [[!
 platform supports activation, the user agent must (unless otherwise specified) run these steps:
 
 <ol>
+ <li><p>Let <var>action</var> be null.
+
+ <li><p>If one of <var>notification</var>'s <a for=notification>actions</a> was activated by the end
+ user, then set <var>action</var> to that <a for=/>notification action</a>.
+
+ <li><p>Let <var>navigationURL</var> be <var>notification</var>'s
+ <a for=notification>navigation URL</a>.
+
+ <li>
+  <p>If <var>action</var> is non-null, then set <var>navigationURL</var> to <var>action</var>'s
+  <a for="notification action">navigation URL</a>.
+
+  <p class=note>This intentionally makes it so that when a <a for=/>notification action</a>'s
+  <a for="notification action">navigation URL</a> is null, it falls through to the
+  <code>click</code> event, providing more flexibility to the web developer.
+
+ <li>
+  <p>If <var>navigationURL</var> is non-null:
+
+  <ol>
+   <li>
+    <p>Select one of the following two options in an <a>implementation-defined</a> manner:
+
+    <ul>
+     <li><p><a for=/>Navigate</a> an existing <a for=/>top-level traversable</a> within the user
+     agent's <a for="user agent">top-level traversable set</a> to <var>navigationURL</var>.
```

Comment:
But does Safari actually focus-and-navigate a tab? I think it just focuses a tab and does not navigate per what I see for the non-push external clicks. But I may be wrong.

Also does this need to say about focusing?

Reply:
It definitely navigates for me when follow a link from a different app or run `https://github.com/whatwg/notifications/pull/213`. Just lost my initial comment trying that.